components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -592,17 +594,17 @@ void AdsServiceImpl::MaybeStart(
     return;
   }
 
-  if (should_restart) {
-    VLOG(1) << "Restarting ads service";
-    Shutdown();
-  }
+  if (IsEnabled()) {
```

Comment:
Maybe we change this to

```
if (IsEnabled()) {
  return;
}
```

so it isn't as nested?

Reply:
Removed the pyramid of doom